code/graphics/post_processing.cpp

```diff
@@ -47,7 +47,17 @@ void parse_lightshafts_func()
 	Post_processing_enable_lightshafts = enabled;
 }
 
-static auto LightshaftsOption __UNUSED = options::OptionBuilder<bool>("Graphics.Lightshafts",
+// used by In-Game Options menu
+bool Post_processing_enable_sunglare = true;
+
+void parse_sunglare_func()
+{
+	bool enabled;
+	stuff_boolean(&enabled);
+	Post_processing_enable_sunglare = enabled;
```

Comment:
Any reason not to parse the boolean directly into `Post_processing_enable_sunglare`?